code/game/objects/items/food/meat.dm

```diff
@@ -540,10 +540,10 @@
 	. = ..()
 	AddComponent(/datum/component/slippery, 30)
 
-/obj/item/food/lasagna
-	name = "Lasagna"
-	desc = "A slice of lasagna. Perfect for a Monday afternoon."
-	icon_state = "lasagna"
+/obj/item/food/timballo
+	name = "Timballo"
+	desc = "A slice of timballo. Perfect for a Monday afternoon."
```

Comment:
is the garfield reference still worthy of staying on non-lasagna?

Reply:
right, i'll have to change that too, since Garfield eats lasagna